### ISSUE TITLE
Atypical perifissural nodules

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6640]
**Preferred term label:** Atypical perifissural nodules

**Synonyms**
Atypical PFN

**Definition (free text, please give PubMed ID)**
Perifissural nodules are typical if they do not have contact with interlobar septum.

**Parent term (use hpo.jax.org/app)**
Perifissural nodules (once created)

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
